frontend/appflowy_flutter/lib/plugins/database/application/cell/bloc/relation_row_search_bloc.dart

```diff
@@ -45,12 +58,56 @@ class RelationRowSearchBloc
 
   void _updateFilter(String filter, Emitter<RelationRowSearchState> emit) {
     final rows = [...allRows];
+
     if (filter.isNotEmpty) {
       rows.retainWhere(
-        (row) => row.name.toLowerCase().contains(filter.toLowerCase()),
+        (row) =>
+            row.name.toLowerCase().contains(filter.toLowerCase()) ||
```

Comment:
Adding brackets to wrap the conditions would make it clearer.

`( A || B) && C`

Reply:
If so, the previous condition is wrong. Because these two operators are executed from left to right.